src/index.css

```diff
@@ -1,3 +1,11 @@
+:root {
+  /* Colors */
+  --skeleton: rgba(125, 125, 125, 0.08);
+
+  /* Animations */
+  --pulse: pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
```

Comment:
nit: I don't think the `--pulse` animation should be set on `root`

Reply:
In case we create more skeletons for different components, the animation for all should be the same so I created it here. 

Same for colors, we should create our variables and use them in the different CSS files instead of hard coding the colors. This would be a better approach.

For now, just moved to it's own folder